nimble/host/src/ble_uuid.c

```diff
@@ -33,21 +33,12 @@ static uint8_t ble_uuid_base[16] = {
 };
 
 #if MYNEWT_VAL(BLE_HS_DEBUG)
-static int verify_uuid(const ble_uuid_t *uuid)
-{
-    switch (uuid->type) {
-    case BLE_UUID_TYPE_16:
-    case BLE_UUID_TYPE_32:
-        return 0;
-    case BLE_UUID_TYPE_128:
-        if (memcmp(BLE_UUID128(uuid)->value, ble_uuid_base, 12) != 0) {
```

Comment:
since `verify_uuid` was under `BLE_HS_DEBUG` I'm not sure why we want to remove this `memcmp`.
Maybe instead of removing it we could print some error? It would be nice to catch that remote behaves incorrectly when sending Bluetooth UUIDs as 128bits value.

Reply:
if remote behaves incorrectly it will just fail and you will find the root cause in logs anyway

Reply:
that is fine then